iredis/client.py

```diff
@@ -58,9 +59,16 @@ def __str__(self):
             return f"{self.host}:{self.port}[{self.db}]"
         return f"{self.host}:{self.port}"
 
-    def execute_command(self, completer, command_name, *args, **options):
+    def execute_command_and_read_response(self, completer, command_name, *args, **options):
         "Execute a command and return a parsed response"
         # === pre hook ===
+        # TRANSATION state chage
+        if command_name.upper() == "MULTI":
```

Comment:
So this place is replaced by an implementation similar to my previous one, without using the callback mechanism to update the config

Reply:
Yes.

Sorry I didn't make this very clear. One command a callback is kind of redundant, so let's put the common render to `renders.py` functions, and put the state migrate to here(After hook or pre hook section).

Sorry to waste your work.

Reply:
In case I didn't make myself clear, let's put the completers update and the common renders(like render simple-string-reply, render int, list etc) to renders.py, and put special hooks like config state change here.

Sorry again about changing the design. :)